libs/designsystem/modal/src/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -144,6 +145,7 @@ export class ModalWrapperComponent
     .asObservable()
     .pipe(debounceTime(this.VIEWPORT_RESIZE_DEBOUNCE_TIME));
   private _mutationObserver: MutationObserver;
+  private mutationObserverDisconnectFn: () => void;
```

Comment:
Consider naming - IMO it's an implementation detail what the return function actually does (unobserve, disconnect or something else)
```suggestion
  private unobserveFn: () => void;
```

Reply:
I think the readability suffers if we omit the implementation details, as we have multiple observers going on in the code. 

We end up with something like this in the destroy hook, which is why I named in the way I did: 
  ```
ngOnDestroy() {
  ...
    this.unobserveFn();
    this.intersectionObserver.disconnect();
    delete this._intersectionObserver;
    if (this.resizeObserverService) {
      this.resizeObserverService.unobserve(this.windowRef.nativeWindow.document.body);
      this.resizeObserverService.unobserve(this.ionHeaderElement.nativeElement);
      this.resizeObserverService.unobserve(this.currentFooter);
    }
   ...
  }
```